VideoWeb/VideoWeb/ClientApp/src/app/shared/header/header.component.spec.ts

```diff
@@ -35,4 +36,8 @@ describe('HeaderComponent', () => {
         component.selectMenuItem(0);
         expect(router.navigate).toHaveBeenCalledWith([component.topMenuItems[0].url]);
     });
+
+    it('has a property for logout route', () => {
+        expect(component.logoutRoute).toBe(pageUrls.Logout);
+    });
```

Comment:
I don't think this test really adds anything. It doesn't look like there's any test for when the link is clicked, it would be good to get one for that and ensure the route is correct that way.